src/Microsoft.ML.AutoML/TrainerExtensions/SweepableParams.cs

```diff
@@ -115,6 +117,15 @@ public static IEnumerable<SweepableParam> BuildLightGbmParams()
             };
         }
 
+        public static IEnumerable<SweepableParam> BuildMatrixFactorizationParams()
+        {
+            return new SweepableParam[]
+            {
+                new SweepableDiscreteParam(nameof(MatrixFactorizationTrainer.Options.NumberOfIterations), new object[] { 10, 20, 40 }),
+                new SweepableFloatParam(nameof(MatrixFactorizationTrainer.Options.LearningRate), 0.001f, 0.1f, numSteps: 2),
+                new SweepableDiscreteParam(nameof(MatrixFactorizationTrainer.Options.ApproximationRank), new object[] { 8, 16, 64, 128 }),
```

Comment:
I took the values here from MatrixFactorizationTrainer
@justinormont could you please cross check if this is OK?

Reply:
Added more sweepable params.